components/bl-gauge-chart-component/src/index.js

```diff
@@ -0,0 +1,46 @@
+import { decorateNumber } from './decorate-number';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function GaugeChartComponent({ component }) {
+  const { classList, display, disable, goal, progress } = component;
+
+  const percentOfProgress = (progress / (goal / 100)).toFixed(2);
```

Comment:
Also, how about move all this calculation into hook 'useChartData' or somethink like this.

In this case component became more light in first look.

`const { progressAngle, progressPercentage, shownProgress, shownGoal } = useChartData(goal, progress)`